src/sentry/tasks/deletion.py

```diff
@@ -377,6 +382,22 @@ def delete_organization_integration(object_id, transaction_id=None, actor_id=Non
         organization_id=instance.organization_id, integration_id=instance.integration_id
     ).update(integration_id=None)
 
+    # delete the identity attached through the default_auth_id
+    if instance.default_auth_id:
+        log_info = {
+            "integration_id": instance.integration_id,
+            "id": instance.default_auth_id,
```

Comment:
is the `id` here the `external_id`? wondering if we should be more specific about what the 'id' is in the logs, maybe even should be the `user_external_id`

Reply:
@MeredithAnya It's the ID of the identity object we are deleting (not the `external_id`). We can call it `identity_id` if that makes more sense to you.

Reply:
@scefali ah okays yeah I guess I was confused, so `identity_id` is good :)